config/routes.rb

```diff
@@ -7,6 +7,10 @@
   resources :users, only: [:new, :create]
   resources :orders, only: [:index, :show, :create]
 
+  namespace :admin do
```

Comment:
I like that you'r sticking to the Stories instead of creating additional routes that aren't necessary yet.